core/src/main/java/com/avast/bytes/Bytes.java

```diff
@@ -21,6 +20,27 @@
  */
 public interface Bytes {
 
+    /**
+     * Hidden class with private fields and methods. The only reason for an existence of this class is
+     * that Interfaces in jdk8 don't support private methods and fields
+     */
+    class Hidden {
```

Comment:
I would just move this to some `Utils` with `static` method, it will be simpler.